src/tests/AccountCard.test.js

```diff
@@ -17,26 +20,116 @@ const props = {
 }
 
 describe('AccountCard test', () => {
```

Comment:
The `describe` block should have all test**s** related to a particular functionality. You should either add an "s" to "test" or write "suite" instead.